src/utils/consts.ts

```diff
@@ -65,19 +68,20 @@ import { ConnectConfig, keyStores } from "near-api-js";
 import { AptosNetwork } from "./aptos";
 import { getNetworkInfo, Network } from "@injectivelabs/networks";
 import { ChainId as InjectiveChainId } from "@injectivelabs/ts-types";
+import { ChainConfiguration } from "@sei-js/react";
 import {
   testnetConnection,
   localnetConnection,
   Connection,
 } from "@mysten/sui.js";
 
 export type Cluster = "devnet" | "testnet" | "mainnet";
-export const CLUSTER: Cluster =
-  process.env.REACT_APP_CLUSTER === "mainnet"
-    ? "mainnet"
-    : process.env.REACT_APP_CLUSTER === "testnet"
-    ? "testnet"
-    : "devnet";
+export const CLUSTER: Cluster = "testnet" as Cluster;
```

Comment:
Why was this change made?

Reply:
This was to force testnet on the Cloudflare preview page that was generated for open PRs (before Sebas implemented the testnet preview along the mainnet preview replacing cloudflare for github actions)

It will be reverted no doubt on following commits before the merge 👌🏼